ajax/libs/expect.js/package.json

```diff
@@ -1,9 +1,26 @@
 {
   "name": "expect.js",
-  "filename": "expect.min.js",
-  "version": "0.2.0",
+  "version": "0.3.1",
+  "license": "MIT",
   "description": "BDD style assertions for node and the browser.",
+  "filename": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Automattic/expect.js.git"
+  },
   "keywords": [
     "expect.js"
```

Comment:
Could you please also enrich the `keywords`? Like `BDD-style`, `should.js`. Thanks.